upup/pkg/fi/cloudup/awstasks/iamrolepolicy.go

```diff
@@ -121,6 +158,51 @@ func (_ *IAMRolePolicy) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *IAMRoleP
 		return fmt.Errorf("error rendering PolicyDocument: %v", err)
 	}
 
+	// Handles the full lifecycle of Policy Overrides
+	if e.Managed {
```

Comment:
I guess Managed is more "ManageAttachments"?

Reply:
I was using this so I didn't have to check if the slice was nil and then check it's length. It just looked cleaner.